frigate/util.py

```diff
@@ -888,7 +889,7 @@ def vainfo_hwaccel() -> sp.CompletedProcess:
 
 def get_ffmpeg_arg_list(arg: Any) -> list:
     """Use arg if list or convert to list format."""
-    return arg if isinstance(arg, list) else arg.split(" ")
+    return arg if isinstance(arg, list) else shlex.split(arg)
```

Comment:
@NickM-27 you may want to re-review this PR after this change.

Reply:
yeah I saw that, can you add a test to the test_ffmpeg_presets.py which manually sets the inputs args as default args but in a single string?

Reply:
Done, I'm even testing the string with an `"arg with space"` in it as well.